components/server/src/prometheus-metrics.ts

```diff
@@ -368,3 +368,13 @@ export type SpiceDBCheckConsistency =
 export function incSpiceDBRequestsCheckTotal(consistency: SpiceDBCheckConsistency) {
     spicedbCheckRequestsTotal.labels(consistency).inc();
 }
+
+export const authorizerSubjectId = new prometheusClient.Counter({
```

Comment:
where is it used an registered? :thinking:

Reply:
It does not need to be. But I just noticed that we are very inconsistent with how we do it here :confused: 
Anyway, found a mistake when verifying: incremented the wrong counter :see_no_evil: